gammapy/data/data_store.py

```diff
@@ -455,6 +457,336 @@ def check(self, checks="all"):
         checker = DataStoreChecker(self)
         return checker.run(checks=checks)
 
+    ##########################################################################################
+    # My contribution:
+
+    def _obscore_def(self):
+        """Generate the Obscore default table
+
+        In case the obscore standard changes, this function should be changed consistently
```

Comment:
@QRemy  Thank you for your comments.

In the next commit you will see many changes, most of them following Regis's suggestions.
Regarding this question, for now we must support one version, and maybe in the future this can evolve, anyway I need to define the bases somewhere and I decided to compact it in a "_obscore_def" function.